### ISSUE TITLE
Release WSS on NI Labview Tools Network

### ISSUE THREAD

**Issue by McGeifer** [MediaMongrels-Ltd/LabVIEW-Secure-WebSockets-Addon#2]
I wonder why this Add-On is not listed in the NI LabVIEW Tools Network? Are there any plans to do so?

**Comment by samsharp99** [comment]
Dear @McGeifer, it is a valid point. At the time I created the package I did not have a lot of time to test it nor do the additional documentation required to submit it to the NI Tools Network. I may consider publishing it to VIPM.io instead as an alternative. If you would like to help add the necessary documentation, examples etc. required by the NI tools network then I would be grateful for the contribution and will allow me to begin the submission process with NI.
